server/src/main/java/org/elasticsearch/action/admin/cluster/node/stats/NodesStatsRequest.java

```diff
@@ -33,6 +34,7 @@ public class NodesStatsRequest extends BaseNodesRequest<NodesStatsRequest> {
 
     private CommonStatsFlags indices = new CommonStatsFlags();
     private final Set<String> requestedMetrics = new HashSet<>();
+    private boolean needShardsStats = true;
```

Comment:
Naming nit/suggestion:

```suggestion
    private boolean includeShardsStats = true;
```

Reply:
done. Fixed in 373cc6f6bcbce124e4036555b20af5598f1d3d4b